client/dashboard/profile-wizard/style.scss

```diff
@@ -27,6 +27,10 @@
 			max-width: 100%;
 			justify-content: center;
 		}
+
+		.business-competitors .woocommerce-select-control__listbox {
+			max-height: 504px;
```

Comment:
This works, but what do you think about using `unset` instead of specifying a height?

Reply:
Good idea! I changed it in the commit [c79716e](https://github.com/woocommerce/woocommerce-admin/pull/4111/commits/c79716e5970f28415cf0d6775de5e1baab27814b).
Thanks!